### DISCUSSION TITLE
Is there any confidence score for Entity Linker?

### DISCUSSION THREAD

**Discussion by amirj** [explosion/spaCy#8031] [Help: Coding & Implementations] [answered]
After adding an entity_linker in a pipeline, we can leverage "ent.kb_id_" to access the corresponding ID. I've been thinking is there any confidence/similarity score associated with this assignment?

**Comment by svlandeg** [comment, marked as answer]
This isn't currently supported no. I agree it would be a useful feature to have. We can keep this in mind for future improvements!
